src/hooks/usePncWebSocket.ts

```diff
@@ -0,0 +1,81 @@
+import { useEffect } from 'react';
+
+import * as pncWebSocketService from 'services/pncWebSocketService';
+
+/**
+ * HOOK CHECKERS
+ */
+
+/**
+ * Additional filtering parameters
+ */
+interface IOnParameters {
```

Comment:
`IOnParameters` sound general but these parameters seem to be related to Build process.

Reply:
`IOnBuildParameters`

Reply:
I renamed it.